src/app/redux/common/index.ts

```diff
@@ -0,0 +1,6 @@
+import commonReducer from './reducers';
+export default commonReducer;
+
+// TODO: Discuss how would we like our directory structure to look for Redux? The redux docs have a 3 recommended approaches:
```

Comment:
I think if we plan to try the `createSlice` approach provided by redux-toolkit, that lends itself to the "ducks" approach. The question is where do our ducks go. Do we want to have all the redux stuff in one place, and have e.g. `src/app/redux/plans/` and `src/app/redux/mappings/` etc, each containing maybe `reducer.ts` and `types.ts`? Maybe we just have `src/app/redux/plans.ts` with the slice and types and everything in one place?

Or maybe we pair the redux code with the relevant react code? We already have e.g. `src/app/Plans/` and `src/app/Mappings/`, which already each contain a `types.ts` file. We could just add `src/app/Plans/reducer.ts` or `src/app/Plans/redux/*` or whatever. The only issue with that is if we end up having more than one "duck" related to each page, or more than one page relating to the same "duck". I guess they could import across those boundaries.

I'm split between these two, leaning towards the `src/app/redux/` folder with each slice/resource-type/duck or whatever in there. I do think we should probably move a lot of our `types.ts` stuff into that area, at least the types that describe API objects. Maybe the types that are specific to UI components can stay near them.

Reply:
I think I like keeping the redux stuff separate & following the ```src/app/redux/```. I don't think it really did us any favors trying to cram the redux stuff into the feature component directories on MTC. Having somewhere to store the api types globally would be a plus to that approach also.